### ISSUE TITLE
Anotation Indentation Issue

### ISSUE THREAD

**Issue by Serubin** [Gillespie59/eslint-plugin-angular#529]
Quick question about indentation and annotations with angular 2:

In a block like this:
```js
@NgModule({
    declarations: [
        ...
    ],
    imports: [
        ...
    ],
    providers: [],
    bootstrap: [ ... ]
})
```

I get indentation errors that basically want the block formatting like this:
```js
@NgModule({
    declarations: [
    ...
    ],
    imports: [
    ...
    ],
    providers: [],
    bootstrap: [ ... ]
    })
```
I think this is an issue with the default eslint indentation parsing, however I'm curious if anyone knows a good work around or if this is different issue.

Thanks!

**Comment by Gillespie59** [comment]
I think you do not use the eslint-plugin-angular for your angular application. The ESLint plugin is specific to AngularJS application.

**Comment by Serubin** [comment]
According to the [README](https://github.com/Gillespie59/eslint-plugin-angular/blob/master/README.md#rules-specific-for-angular-1-or-2) this should support angular2, no?

**Comment by Gillespie59** [comment]
Yes but we do not have any rules specific to angular
